src/main/java/org/jabref/gui/importer/actions/OpenDatabaseAction.java

```diff
@@ -158,14 +168,14 @@ public void openFiles(List<File> filesToOpen, boolean raisePanel) {
         // Run the actual open in a thread to prevent the program
         // locking until the file is loaded.
         if (!filesToOpen.isEmpty()) {
-            final List<File> theFiles = Collections.unmodifiableList(filesToOpen);
+            final List<Path> theFiles = Collections.unmodifiableList(filesToOpen);
             JabRefExecutorService.INSTANCE.execute(() -> {
-                for (File theFile : theFiles) {
+                for (Path theFile : theFiles) {
                     openTheFile(theFile, raisePanel);
                 }
             });
-            for (File theFile : theFiles) {
-                frame.getFileHistory().newFile(theFile.getPath());
+            for (Path theFile : theFiles) {
+                frame.getFileHistory().newFile(theFile.toString());
```

Comment:
I'm always confused by this: is `Path.toString` or `Path.toAbsolutePath.toString` the correct replacement for `File.getPath`?